packages/dev/core/src/Gizmos/boundingBoxGizmo.ts

```diff
@@ -358,6 +358,11 @@ export class BoundingBoxGizmo extends Gizmo {
 
                         // Rotate around center of bounding box
                         this._anchorMesh.addChild(this.attachedMesh, Gizmo.PreserveScaling);
+                        if (this._anchorMesh.getScene().useRightHandedSystem) {
+                            this._tmpQuaternion.x *= -1;

```

Comment:
Please use https://github.com/BabylonJS/Babylon.js/blob/5406aca3f55afdda974600032c21af71aae4190a/packages/dev/core/src/Maths/math.vector.ts#L3688

Reply:
Thanks for you guidance, I replaced it with conjugateInPlace() method :)